FunctionalTests/FunctionalTest1.cs

```diff
@@ -16,49 +16,44 @@ public FunctionalTest1(ITestOutputHelper testOutputHelper)
     {
         _testOutputHelper = testOutputHelper;
     }
-
-    private void KillBackend()
-    {
-        _proc?.Kill(true);
-        _proc = null;
-    }
     
-    private void StartBackend()
+    [Fact]
+    public async Task TestApiTest()
     {
-        KillBackend();
-        
-        // Execute backend project using dotnet.
-        var proc = new Process
-        {
-            StartInfo = new ProcessStartInfo
-            {
-                FileName = "dotnet",
-                Arguments = $"run --project {BackendPath}",
-                UseShellExecute = true,
-                CreateNoWindow = true,
-            }
-        };
+        const string endpoint = $"{ApiPath}/test/helloWorld";
+        const string expected = "Hello, world!\n";
+        _testOutputHelper.WriteLine(endpoint);
 
-        var started = proc.Start();
+        StartBackend();
+        
+        // Allow 5 seconds for backend to start.
+        await Task.Delay(5_000);
 
-        if (started is false)
-            throw new Exception("Could not start backend.");
+        await HitEndpoint(endpoint, expected);
 
-        _proc = proc;
+        KillBackend();
     }
-    
+
     [Fact]
-    public async Task TestApiTest()
+    public async Task InventorySizeTest()
```

Comment:
Test that just checks inventory size to be 0.

Once we have real endpoints to test, remove.

Reply:
Also, there is a conversation to be had about whether or not these should just be unit tests.  I think they *should* because they are really just testing a singular endpoint.

However, I opted to make them functional tests because well, I want to verify the whole application is actually using the database and working correctly.